### ISSUE TITLE
Reviewer replies should be logged in add-ons review log

### ISSUE THREAD

**Issue by abhn** [mozilla/addons-server#21128]
### Summary
On the `/reviewers/reviewlog` page, we currently list reviewer actions like approval, confirming auto-approval, rejection and scheduled rejections. 

We do not log reviewer replies on this page

### Proposal
- Log all reviewer replies in review log just like we do for approvals and rejections (with the content of the reply message searchable as well)

### Usecase
- Reviewers frequently look at previously handled similar cases to draft their response by searching for a keyword. Being able to search through the complete list of responses would make the search more thorough.
- Reviewers need a way to look at / find an unlisted auto-approved add-on that they previously processed and if the only action taken on it was a reviewer reply, it is hard to find it again. 

### Anything else we should know?
- The search functionality on review log page is very slow (especially for longer time span searches), and while that's an existing issue, could it amplify if we make more things searchable?

**Comment by wagnerand** [comment]
There is also a use case to only have decisive actions (the ones we include at the moment) in the log, so we might need a switch or similar to toggle showing decisive vs all.

**Comment by abhn** [comment]
@wagnerand yep, that sounds good. The reviewer replies with canned responses are theoretically decisive actions, but I'd guess we do not have any consistent way of differentiating them from the other reviewer replies? In that case a toggle to exclude replies from search would be nice.

**Comment by wagnerand** [comment]
@abhn Instead of patching the workaround for unlisted even further, we should work towards addressing the root cause, i.e. looking into what we can do around a rejection mechanism for auto-approved unlisted files.

**Comment by eviljeff** [comment]
> Instead of patching the workaround for unlisted even further, we should work towards addressing the root cause, i.e. looking into what we can do around a rejection mechanism for auto-approved unlisted files.

This appears very different than the issue reported here.  If there isn't an issue/ticket to address the deficiencies can one be logged ?  (and close this issue if there is a preferred fix)

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-479
